src/share/cxx/prim_driver.cpp

```diff
@@ -64,27 +65,21 @@ void prim_run_subcycle_c (const Real& dt, int& nstep, int& nm1, int& n0, int& np
 
   tl.update_tracers_levels(params.qsplit);
 
-  // Apply forcing
-#ifdef CAM
-  Errors::runtime_abort("CAM forcing not yet availble in C++.\n",
-                        Errors::err_not_implemented);
-  // call TimeLevel_Qdp(tl, qsplit, n0_qdp)
-
-  // if (ftype==0) then
-  //   call t_startf("ApplyCAMForcing")
-  //   call ApplyCAMForcing(elem, hvcoord,tl%n0,n0_qdp, dt_remap,nets,nete)
-  //   call t_stopf("ApplyCAMForcing")
-
-  // elseif (ftype==2) then
-  //   call t_startf("ApplyCAMForcing_dynamics")
-  //   call ApplyCAMForcing_dynamics(elem, hvcoord,tl%n0,dt_remap,nets,nete)
-  //   call t_stopf("ApplyCAMForcing_dynamics")
-  // endif
-
-#else
+#ifndef CAM
   apply_test_forcing ();
 #endif
 
+  // Apply forcing.
+  // In standalone mode, params.ftype == ForcingAlg::FORCING_OFF
+  // Corresponds to ftype == 0 in Fortran
```

Comment:
why not call it ftype0 then? i don't know how the code looked like when we branched, but now forcing is always 'ON', but how it works depends on ftype and tests.

Reply:
that is, i believe there should not be FORCING_OFF at all, and _DEBUG should be called ftype0?